crates/ide-db/src/search.rs

```diff
@@ -134,6 +134,7 @@ pub enum ReferenceCategory {
     // FIXME: Some day should be able to search in doc comments. Would probably
     // need to switch from enum to bitflags then?
     // DocComment
+    Test,
```

Comment:
Something can be a test and a read at the same time. So, the time has come to use a border here! This is *precisely* the situation the old comment just above this line talks about